hathor/daa.py

```diff
@@ -41,6 +41,8 @@
 
 
 class TestMode(IntFlag):
+    __test__ = False
```

Comment:
I didn't understand what it is doing and why it fixes the warning. Do you have any documentation to link here? Maybe add as a comment in the code?

Reply:
Never mind. I got it.